po/be.po

```diff
@@ -158,162 +158,161 @@ msgstr ""
 
 #: src/home/org.freedesktop.home1.policy:94
 #, fuzzy
-#| msgid "Authentication is required to manage system services or other units."
 msgid "Authentication is required to manage signing keys for home directories."
 msgstr ""
 "Для кіравання сістэмнымі службамі і іншымі адзінкамі патрабуецца "
 "аўтэнтыфікацыя."
```

Comment:
Hmm, not your fault, but this is another place where we're distributing a bogus translation. We need to shut that fuzzy shit down.